compareECLFiles.cmake

```diff
@@ -12,6 +22,8 @@ set(BASE_RESULT_PATH ${PROJECT_BINARY_DIR}/tests/results)
 # Input:
 #   - casename: basename (no extension)
 #
+# Details:
+#   - This test class compares output from a simulation to reference files.
 macro (add_test_compareECLFiles casename filename simulator)
```

Comment:
we need to group these because we do not want to add multiple ctest entries. the driver is the grouping - run simulation, then do appropriate comparisons. if i split we end up with dependencies between tests and all the mess that leads to. it's a design criteria to avoid this.